hs_core/models.py

```diff
@@ -38,7 +39,7 @@
 from mezzanine.conf import settings as s
 from mezzanine.pages.managers import PageManager
 
-from dominate.tags import div, legend, table, tbody, tr, th, td, h4
+from dominate.tags import div, legend, table, tbody, tr, th, td, h4, br
```

Comment:
when and why did we start using dominate?  I know it's been around for awhile, but i'm curious why we started using it.

Reply:
I find it useful for dynamically generating html snippets from python. Otherwise, I guess we would have to generate these html snippets in JavaScript. I preferred to do it in python as it was easier for me to do it in python vs JS.

Reply:
I guess I don't see why we couldn't just use django templates to create the snippets.  For example, couldn't the get_html method on line 1355 that was modified below just have two templates setup with that html structure, then you could just return the appropriate template based on the logic on type and pass in the context?

I definitely don't want pure JS.  @cuahsimarko I looked into jinja2 a little bit and it seems to be just a newer version of Django template... It's not clear to me what it would add.

I like the looks of dominate as a developer that doesn't spend much time in the frontend... however I think it's less appealing to frontend devs like @Maurier and @engineSound and I'm worried it might cause some bleeding between the front and back ends.

Reply:
@Maurier and @engineSound As you can see @sblack-usu has raised a valid concern that front end developers may not feel comfortable the way I am generating html code using python. All file type/aggregation level metadata related html is generated in python. If you prefer that we should be using Django template to write the html code for filetype metadata then I would suggest we create a new issue for this conversion and one of you can do this conversion. I would be happy to help you in that conversion.